.github/workflows/snyk-pr.yml

```diff
@@ -27,5 +27,5 @@ jobs:
         PR_TITLE: ${{ github.event.pull_request.title }}
       with:
         jira_token: ${{ secrets.JIRA_TOKEN_PUBLIC_REPO }}
-        gh_token: ${{ secrets.GITHUB_TOKEN }}
+        gh_token: ${{ secrets.KAFKA_GITHUBAPP_TOKEN }}
```

Comment:
Hi @sfc-gh-jfan I dont see this running at all, do we need this workflow file? Can you confirm? 
I only see